pkg/cluster/clusterLeader.go

```diff
@@ -295,12 +295,23 @@ func (cluster *Cluster) StartLeaderCluster(c *kubevip.Config, sm *Manager, bgpSe
 						}
 
 						for {
+
+							// Ensure the address exists on the interface before attempting to ARP
+							set, err := cluster.Network.IsSet()
+							if err != nil {
+								log.Warnf("%v", err)
+							}
+							if !set {
+								err = cluster.Network.AddIP()
+								log.Warnf("%v", err)
```

Comment:
nit: should we check if err != nil? I.e.
```
err = cluster.Network.AddIP()
if err != nil {
    log.Warnf("%v", err)
}
```

Reply:
We should! How did I miss that! 🙄

Fix incoming. 😀